styles.css

```diff
@@ -113,9 +113,52 @@ button {
     width: auto;
 }
 
-@media screen and (max-width: 300px) {
+@media screen and (max-width: 600px) {
```

Comment:
make sure that this is the actual size that you want the screen to change and not an arbitrary value.

Reply:
Yes! I was messing with it when my media queries weren't working. Thank you for trouble shooting that with me. Meta tag!